Makefile

```diff
@@ -35,11 +35,16 @@ SEED_NAME                                  := ""
 DEV_SETUP_WITH_WEBHOOKS                    := false
 KIND_ENV                                   := "skaffold"
 PARALLEL_E2E_TESTS                         := 5
+SUDO                                       :=
 
 ifneq ($(strip $(shell git status --porcelain 2>/dev/null)),)
 	EFFECTIVE_VERSION := $(EFFECTIVE_VERSION)-dirty
 endif
 
+ifneq (, $(shell which sudo))
+	SUDO := sudo -E
+endif
```

Comment:
Does this mean that `make kind-down` etc. now always require `sudo` privileges?

Reply:
Yes. This is because etcd backup restore creates files with the root user and no permissions are given to the group or others.

Probably the backup restore can be enhanced to provide `rw` permissions also to the group of the files and then SETGUID bit should be sufficient. Or maybe there is even a bug that after shoot deletion the backup files are not deleted from the "bucket"?

Reply:
I think we can remove permissions elevation if provider local implements `Delete` for backupbucket and backupentries

https://github.com/gardener/gardener/blob/3ef1f2453b81e4ad90ec358c7d5191cb1bed3643/pkg/provider-local/controller/backupentry/actuator.go#L50-L52

https://github.com/gardener/gardener/blob/3ef1f2453b81e4ad90ec358c7d5191cb1bed3643/pkg/provider-local/controller/backupbucket/actuator.go#L58-L60

What do you think?

Reply:
I'm not into the details, but it definitely sounds like a promising option.

Reply:
@plkokanov do you know if there is some special requirement so that the provider local intentionally has not implemented deletion of the backupbuckets and backupentries directories?

Reply:
I've implemented the `Delete` methods. Now the backup bucket and entries directories are removed by provider-local. 
As etcd-backup-restore runs as root, provider-local also needs to run as root, otherwise it cannot delete the files.